### ISSUE TITLE
The default size buf was used to read the command execution result, resulting in an incomplete command execution result.

### ISSUE THREAD

**Issue by Li4n0** [appleboy/easyssh-proxy#70]
```golang
func (ssh_conf *MakeConfig) Stream(command string, timeout ...time.Duration) (<-chan string, <-chan string, <-chan bool, <-chan error, error) {
	... ...
	// combine outputs, create a line-by-line scanner
	stdoutReader := io.MultiReader(outReader)
	stderrReader := io.MultiReader(errReader)
	stdoutScanner := bufio.NewScanner(stdoutReader)
	stderrScanner := bufio.NewScanner(stderrReader)
	... ...
```

The default buf size is 4096. This results in an incomplete output when the command stdout or stderr is greater than this value

**Comment by appleboy** [comment]
@Li4n0 Yes

**Comment by Li4n0** [comment]
> @Li4n0 Yes
So will this problem be solved?

**Comment by fatelei** [comment]
how about add bufsize config？

**Comment by appleboy** [comment]
fixed in #78
